translations/system-configuration.tr.xliff

```diff
@@ -0,0 +1,102 @@
+            </trans-unit>
+            <trans-unit id="form_user">
+                <source>form_user</source>
+                <target state="translated">Kullanıcı—varsayılan değerler</target>
+            </trans-unit>
+            <trans-unit id="theme">
+                <source>theme</source>
+                <target state="translated">Tema</target>
+            </trans-unit>
+            <trans-unit id="calendar">
+                <source>calendar</source>
+                <target state="translated">Takvim</target>
+            </trans-unit>
+            <trans-unit id="label.theme.markdown_content">
+                <source>label.theme.markdown_content</source>
+                <target state="translated">Yorum ve açıklamalarda markdown formatına izin ver</target>
+            </trans-unit>
+            <trans-unit id="label.timesheet.mode">
+                <source>label.timesheet.mode</source>
+                <target state="translated">Zaman takip modu</target>
+            </trans-unit>
+            <trans-unit id="label.timesheet.mode_default">
+                <source>label.timesheet.mode_default</source>
+                <target state="translated">[Varsayılan] başlangıç ve bitiş saatleri düzenlenebilir</target>
+            </trans-unit>
+            <trans-unit id="label.timesheet.mode_duration_only">
+                <source>label.timesheet.mode_duration_only</source>
+                <target state="translated">[Süre] bitiş zamanını, süre alanıyla değiştirir</target>
+            </trans-unit>
+            <trans-unit id="label.timesheet.mode_duration_fixed_begin">
+                <source>label.timesheet.mode_duration_fixed_begin</source>
+                <target state="translated">[Süre] ayarlanabilir sabit başlangıç saati, yalnızca süre değiştirilebilir</target>
+            </trans-unit>
+            <trans-unit id="label.timesheet.mode_punch">
+                <source>label.timesheet.mode_punch</source>
+                <target state="translated">[Time-clock] user can start and stop records, but not edit the times or duration</target>
+            </trans-unit>
+            <trans-unit id="label.timesheet.rules.allow_future_times">
+                <source>label.timesheet.rules.allow_future_times</source>
+                <target state="translated">Allow time entries in the future</target>
+            </trans-unit>
+            <trans-unit id="label.timesheet.active_entries.hard_limit">
+                <source>label.timesheet.active_entries.hard_limit</source>
+                <target state="translated">İzin verilen eşzamanlı zaman kaydı sayısı</target>
+            </trans-unit>
+            <trans-unit id="label.timesheet.active_entries.soft_limit">
+                <source>label.timesheet.active_entries.soft_limit</source>
+                <target state="translated">Number of running time entries from which a warning is displayed</target>
```

Comment:
You mean this one?

Reply:
Behause you don't understand the meaning? You are right, that could be improved...
It means: `Number of running timesheet records before a warning is displayed`.

Reply:
I understand but can not find a meaningful translation. Let me think.

Reply:
So, when you set this number to 3 and have two running timesheet records everything is good. But if you start the 3 record, you will see a little warning:
<img width="125" alt="Bildschirmfoto 2019-10-15 um 15 35 25" src="https://user-images.githubusercontent.com/533162/66836321-6bbe2380-ef61-11e9-8cb6-e622968f6bdc.png">
In this example only 1 record is allowed